plugin/k8s_external/xfr.go

```diff
@@ -0,0 +1,94 @@
+package external
+
+import (
+	"context"
+	"net"
+	"time"
+
+	"github.com/coredns/coredns/plugin/pkg/log"
+	"github.com/coredns/coredns/request"
+
+	"github.com/miekg/dns"
+)
+
+const transferLength = 2000
+
+// Serial implements the Transferer interface.
+func (e *External) Serial(state request.Request) uint32 { return uint32(time.Now().Unix()) }
```

Comment:
In your PR description you say this is "smart", here it just takes time.Now(). So it's not smart?